frontend/src/components/deployments/display-code/DisplayCode.jsx

```diff
@@ -46,7 +46,7 @@ const DisplayCode = ({ isDialogOpen, setDialogOpen, url }) => {
         headers = {
           'Authorization': 'Bearer REPLACE_WITH_API_KEY'
         }
-        payload = {'timeout': '80'}
+        payload = {'timeout': 300, 'include_metadata': False}
```

Comment:
Is this timeout too long? I don't have full context here... @Deepak-Kesavan

Reply:
Just now confirmed the same with QA. They mentioned to show the maximum supported timeout which is `300`